README.md

```diff
@@ -1,93 +1,283 @@
+
+Provisional memberships are expected to be temporary, lasting just long enough for contributors from that organization to demonstrate their values and earn representation as individual contributors.
+
+Provisional memberships can also be revoked if and when the Council determines that the organization or the individual is no longer acting in the larger interests of the conda community. The voting procedure for such removals are described in the "[Remove a Provisional Steering Council Member](#remove-a-provisional-steering-council-member)" section below.
+
+Organizations may submit an appeal to the Council to stay their provisional membership if and when the organization determines that the individual is no longer acting in the larger interests of the organization or the conda community. The Council will consider the appeal and follow the same voting procedure as described in the "[Remove a Provisional Steering Council Member](#remove-a-provisional-steering-council-member)" section below. The organization may then nominate a replacement for the provisional membership following the procedure described in in the "[Nominate a Provisional Steering Council Member](#nominate-a-provisional-steering-council-member)" section below.
+
+Finally, **no more than 1/3 of the Steering Council membership can be Provisional Members.** At all times, at least 2/3 of the Steering Council membership hold a seat because of demonstrated contributions to the conda ecosystem. If this threshold is violated then all other Council business is suspended until this requirement is met. The only allowed actions, in this case, are votes on removing Provisional members, votes on adding new contributing members, votes to resolve overlapping funding, existing members moving to Emeritus status, or existing members coming back from Emeritus status.
+
+## Shared Funding Membership restrictions
+
+**No more than 2 council members can have the same funder:**  Each Steering Council member can have the same funding organization as **at most one other Council member.**
+
+The conda Organization [actively encourages](#provisional-members) participation by commercial entities.  However, we also need to preserve the voices of individual contributors, and prevent the conda Organization from ever favoring the financial interests of any commercial entity over the interests of the larger community.
+
+The conda Organization enforces this restriction by tracking the funding sources of each Steering Council member.
+
+### Shared Funding: Proximate, Ultimate, and In Between
+
+Generally, people have shared funding if they are an employee, officer, or member of the Board of Directors of the same entity; if they have a significant consulting relationship with the same entity; or if they own at least 1% of the equity or debt, or derivatives thereof, of the same entity.
```

Comment:
Curious how we came to the 1% equity or debt provision. Think 5% is typically when one needs to make filings about involvement. Also debt doesn't usually give voting rights (except for some peculiar circumstances like restructuring, in which case it is converted to equity). Mentioning this as I'm wondering if this is unnecessarily restrictive on say entrepreneurs for example

Reply:
Good point, @tnabtaf may be able to shed more light on where this came from.

Reply:
@jakirkham I copied this number straight from the PSF governance PEP.  That paragraph was written by Van Lindberg.  I don't have a strong opinion.